subprojects/core/src/main/java/org/gradle/api/internal/file/UnionFileTree.java

```diff
@@ -53,12 +54,17 @@ public void visitContents(FileCollectionResolveContext context) {
     }
 
     @Override
-    public UnionFileTree add(FileCollection source) {
+    public FileCollection add(FileCollection collection) throws UnsupportedOperationException {
+        DeprecationLogger.nagUserOfReplacedMethod("FileTree.add()", "ConfigurableFileTree.from()");
```

Comment:
Isn't the method which is deprecated `FileCollection.add` and not `FileTree.add`?

Reply:
You're too good. Fixed.